test/Common/TestOrchestrations.cs

```diff
@@ -659,6 +659,22 @@ public static async Task<string> LargeEntity([OrchestrationTrigger] IDurableOrch
             return "ok";
         }
 
+        public static async Task<string> EntityIdCaseInsensitivity([OrchestrationTrigger] IDurableOrchestrationContext ctx)
+        {
+            var entityId = ctx.GetInput<EntityId>();
+
+            string content = new string('.', 100000);
+            await ctx.CallEntityAsync<int>(entityId, "set", content);
+
+            var result = await ctx.CallEntityAsync<string>(entityId, "get");
+            if (result != content)
+            {
+                return $"fail: wrong entity state";
+            }
+
+            return "ok";

```

Comment:
Since the test you're writing doesn't really care what the entity does, I suggest you just re-use an existing entity function instead of writing a new one.